README.md

```diff
@@ -165,22 +167,9 @@ Two reasons why Homebridge may not be discoverable:
   
   3. iOS DNS cache has gone stale or gotten misconfigured. Fix: Turn airplane mode on and back off to flush the DNS cache. (This is a temporary fix, but can be repeated when the problem recurs. No permanent fix is as yet known/confirmed. If you're experiencing this as a recurrent issue, it likely affects other bonjour and .local DNS resolution services, for which cycling airplane mode will also temporarily resolve.)
 
-### Errors on startup
-
-The following errors are experienced when starting Homebridge and can be safely ignored. The cost of removing the issue at the core of the errors isn't worth the effort.
-
-```
-*** WARNING *** The program 'nodejs' uses the Apple Bonjour compatibility layer of Avahi
-*** WARNING *** Please fix your application to use the native API of Avahi!
-*** WARNING *** For more information see http://0pointerde/avahi-compat?s=libdns_sd&e=nodejs
-*** WARNING *** The program 'nodejs' called 'DNSServiceRegister()' which is not supported (or only supported partially) in the Apple Bonjour compatibility layer of Avahi
-*** WARNING *** Please fix your application to use the native API of Avahi!
-*** WARNING *** For more information see http://0pointerde/avahi-compat?s=libdns_sd&e=nodejs&f=DNSServiceRegister
-```
-
 ### Limitations
 
- * One installation of Homebridge can only expose 100 accessories due to a HomeKit limit. You can however run multiple Homebridge instances by pointing them to different config and persistence paths (see issue #827).
+ * One installation of Homebridge can only expose 150 accessories due to a HomeKit limit. You can however run multiple Homebridge instances by pointing them to different config and persistence paths (see issue #827).
```

Comment:
HAP-NodeJS is currently set to 150.

https://github.com/KhaosT/HAP-NodeJS/blob/bace3b682fa664993a1d5504ffed42b5c08fed4c/src/lib/Accessory.ts#L36